pancreabble/urchin.py

```diff
@@ -144,7 +145,8 @@ def main(self, args, app):
 
         status = 'openaps @ {:%-I:%M%P}'.format(datetime.now())
         if args.status_text:
-            status = args.status_text
+            status = arg
+            s.status_text
```

Comment:
Please read over your code and fix typos like this before submitting a PR.

This makes me worry that this change has not been tested very thoroughly.

Reply:
fixed